task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -66,8 +73,26 @@ def run(self, task_config: KubernetesTaskConfig) -> None:
                     ),
                 ),
             )
-
-        # TODO(TASKPROC-231): actually launch a Pod
+        # TODO Add volume_devices and volume_mounts
+        container = V1Container(
+            image=task_config.image,
+            name=task_config.name,
+            command=[task_config.command]
+        )
+        pod = V1Pod(
+            metadata=V1ObjectMeta(
+                name=task_config.pod_name,
+                namespace=self.namespace
+            ),
+            spec=V1PodSpec(
+                restart_policy=task_config.restart_policy,
+                containers=[container]
+            ),
+        )
+        self.api.create_namespaced_pod(namespace=self.namespace, body=pod)
```

Comment:
we'll want to use `kube_client` here instead: 
```suggestion
        self.kube_client.core.create_namespaced_pod(namespace=self.namespace, body=pod)
```
so that we can ensure that there's no code path that tries to to k8s api things without loading a kubeconfig first (since creating a KubeClient will automatically do that for us)

Reply:
I think we'll also want to wrap this in a `try-except`, log that the `run()` failed, and return `None` if this failed so that we don't crash Tron should the `run()` call error out in anyway (alternatively, we could probably return True/False if we were able to submit a request to k8s to run our pod since we know the pod name ahead of time anyway :p

Reply:
Added try-except that returns None if error occurs. Also shows the exception we run into.